### DISCUSSION TITLE
Support starting bisect from an unreachable good commit

### DISCUSSION THREAD

**Discussion by fuzy112** [magit/magit#5238] [Ideas]
With recent versions of Git, when a bisection is started and when some good commits are not ancestors of the bad commit, it will compute the merge base and choose the merge base as the first commit to test.  Is it possible for magit to support this use case?

**Comment by tarsius** [comment]
> With recent versions of Git,

I.e., "there's this new feature in Git, which Magit does not support yet".

Based on that, I am treating this as a feature request, and have changed the title.

**Comment by tarsius** [comment]
With fe56c0d10a91d00aed65f831b8435a7224bc7730 assert is no longer needlessly strict. I ended up considering this a bug after all.
